src/std/env.ab

```diff
@@ -64,7 +64,7 @@ pub fun is_command(command: Text): Bool {
 
 /// Creates a prompt and returns the value.
 pub fun input_prompt(prompt: Text): Text {
-    trust $ read -p "\${nameof prompt}" $
+    trust $ read -p "\${nameof prompt} < /dev/tty" $
```

Comment:
Also... shouldn't this line be more like this? I get some weird output when running it as is
```suggestion
    trust $ read -p "\${nameof prompt}" < /dev/tty $
```